src/strand-datepicker/strand-datepicker.js

```diff
@@ -240,6 +244,12 @@
 			if(this.useCommit) this.classList.add('has-footer');
 		},
 
+		_calendarFilter: function(date) {
+			if(date && date !== date.toString()) {
+				return new Date(moment(date).utc().format(this.dateFormat));
```

Comment:
isn't the `.utc()` here a problem (because `moment(new Date("Thu Nov 10 2011 19:00:00 GMT-0600")).format("MM/DD/YYYY") === "11/11/2011"`)

I'm considering a case where the developer passes in an explicit `allowedStart`/`allowedEnd` `Date()` object that is in local time -- this `_calendarFilter` could change the "MM/DD/YYYY" value

(also not so sure about relying on the `.dateFormat` property to have a format that would yield the expected information here -- I'm considering an adversarial user (because I'm paranoid like that)